MAVProxy/modules/lib/wavefront.py

```diff
@@ -0,0 +1,351 @@
+# Copyright (C) 2016  Intel Corporation. All rights reserved.
```

Comment:
I didn't want to impose an extra requirement to the project and writing a parser didn't look hard, so I just decided on writing my own parser.

Anyway, I just tried to install it with `pip install PyWavefront` as it says [here](https://github.com/greenmoss/PyWavefront) with no success.